frontend_vue/src/api/main.ts

```diff
@@ -43,7 +43,7 @@ export function manageToken(params: TokenAuthType) {
     .catch((error) => error.response);
 }
 
-export function downloadToken(params: DownloadTokenType) {
+export function downloadAsset(params: DownloadAssetType) {
```

Comment:
why the name change?

Reply:
This endpoint is not used only for tokens: in this diff I use the same endpoint to download the alert list CSV or JSON

Reply:
maybe we just rename the function to `download` then? we can keep the type?